src/wallet/spend.h

```diff
@@ -52,9 +52,9 @@ class COutput
     /** Whether the transaction containing this output is sent from the owning wallet */
     bool m_from_me;
 
-    COutput(const CWallet& wallet, const CWalletTx& wtx, int iIn, int depth, int input_bytes, bool spendable, bool solvable, bool safe, int64_t time, bool from_me)
```

Comment:
1f475858590a50a8a3f37a02df2c2d7b803f9834: it seems `wallet` was already unused at this point, so you could drop it in another commit. But at least the commit message should mention them both.